src/pages/FeeRateTracker/FeeRateTrackerComp.tsx

```diff
@@ -102,7 +102,7 @@ export const ConfirmationTimeFeeRateChart = ({
     if (!cur.confirmationTime) {
       return acc
     }
-    const range = Math.round(cur.confirmationTime / 10)
+    const range = Math.floor((Math.ceil(cur.confirmationTime) - 1) / 10)
```

Comment:
`cur.confirmationTime` is an integer, so `Math.cell` does nothing here

Reply:
What's the logic of 
```
    const range = Math.floor((Math.ceil(cur.confirmationTime) - 1) / 10)
```

Reply:
> What's the logic of
> 
> ```
>     const range = Math.floor((Math.ceil(cur.confirmationTime) - 1) / 10)
> ```


When confirmationTime is 4 `Math.round(confirmationTime / 10)` returns 0, and When confirmationTime is 5 it returns 1.
Actually it should always returns 0 when confirmationTime is between `0s ~ 10s` (except 0s, 0 was filtered out by the line 102), 
`Math.floor((confirmationTime -1) / 10)` could grouping confirmationTime into `(0 ~ 10], (10 ~ 20], (20 ~ 30] ...` also `[1 ~ 10], [11 ~ 20], [21 ~ 30] ...`

`Math.ceil` is for handling wrong decimal confirmationTime, it can be removed.

Reply:
Got it